src/rubintv/models/models_assignment.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
```

Comment:
This whole file is a pattern I am not used to seeing, which worries me. You have no defined functions or classed, so it's more like a script, and you're importing `cameras` and `locations` which are defined in here. I'm not saying it's necessarily bad, but it's not something I'm familiar with, and it does make me wonder if there are good reasons to avoid this pattern.

Reply:
Hmm... it did feel a little maverick as I was writing it. I'll look up what alternative patterns there are for loading yaml into dataclasses

Reply:
I mean, if these were in `get_x()` functions I wouldn't have blinked, it's just the script-like nature of the file and importing those variable felt a little unusual.